scipy/stats/__init__.py

```diff
@@ -291,16 +312,36 @@
    normaltest
 
 Objects returned by some statistical tests
-==========================================
+------------------------------------------
 
 .. autosummary::
    :toctree: generated/
 
    BinomTestResult
 
 
+Masked statistics functions
```

Comment:
I'm curious - why elevate masked statistics?

Reply:
Because at least it's a coherent set of functionality. The stuff below it is just a random grab bag of areas with a few functions each in them.